march_hardware/src/ethercat/ethercat_master.cpp

```diff
@@ -95,9 +95,9 @@ bool EthercatMaster::ethercatSlaveInitiation(std::vector<Joint>& joints)
 
   for (Joint& joint : joints)
   {
-    if (joint.hasIMotionCube())
+    if (joint.getMotorControllerSlaveIndex() >= 0)
```

Comment:
Actually, a slave index of 0 does not exist, since 0 is the ethercat master
```suggestion
    if (joint.getMotorControllerSlaveIndex() > 0)
```